dbms/src/Client/Benchmark.cpp

```diff
@@ -473,33 +472,10 @@ int mainEntryClickhouseBenchmark(int argc, char ** argv)
 			options["json"].as<std::string>(),
 			settings);
 	}
-	catch (const Exception & e)
-	{
-		std::string text = e.displayText();
-
-		std::cerr << "Code: " << e.code() << ". " << text << "\n\n";
-
-		/// Если есть стек-трейс на сервере, то не будем писать стек-трейс на клиенте.
-		if (std::string::npos == text.find("Stack trace"))
-			std::cerr << "Stack trace:\n"
-				<< e.getStackTrace().toString();
-
-		return e.code();
-	}
-	catch (const Poco::Exception & e)
-	{
-		std::cerr << "Poco::Exception: " << e.displayText() << "\n";
-		return ErrorCodes::POCO_EXCEPTION;
-	}
-	catch (const std::exception & e)
-	{
-		std::cerr << "std::exception: " << e.what() << "\n";
-		return ErrorCodes::STD_EXCEPTION;
-	}
 	catch (...)
 	{
-		std::cerr << "Unknown exception\n";
-		return ErrorCodes::UNKNOWN_EXCEPTION;
+		std::cerr << getCurrentExceptionMessage(true) << std::endl;
```

Comment:
Need to add support for avoiding double stack trace in `getCurrentExceptionMessage`.
Also search for `"Stack trace"` in code base. We need to unify this logic.